app/src/main/java/org/simple/clinic/overdue/AppointmentRepository.kt

```diff
@@ -129,6 +129,17 @@ class AppointmentRepository @Inject constructor(
     )
   }
 
+  fun overdueAppointmentsCount(
+      since: LocalDate,
+      facility: Facility
+  ): Observable<List<Int>> {
+    return overdueDao.overdueAtFacilityCount(
```

Comment:
What do you think of changing the repository method to return `Observable<Int>` instead and map the list by summing the values within the repository method? This way, the abstraction no longer leaks and we can change the query later without affecting consumers of this API.

Reply:
Does this makes sense?
```kt
    return overdueDao.overdueAtFacilityCount(
        facilityUuid = facility.uuid,
        scheduledBefore = since,
        scheduledAfter = since.minus(appointmentConfig.periodForIncludingOverdueAppointments)
    ).map { it.size }
```

Reply:
Yup, looks good.